internal/node/test/test_runfiles_helper.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+const args = process.argv.slice(2);
+const outfile = args.shift();
+const runfiles = require(process.env['BAZEL_NODE_RUNFILES_HELPER']);
+const golden =
```

Comment:
this line looks strange, lots of spaces for some reason?

Reply:
clang format did that for line length I think